.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is this some new yarn thing? Doesn't seem related to the conversion, but I've also not see it in any project before.

Reply:
I found I had to pin yarn to this version using the rc file to get it to work, some bug in yarn 1.21 was preventing me installing any new dependencies: https://github.com/yarnpkg/yarn/issues/7807